mmv1/products/bigquery/Dataset.yaml

```diff
@@ -290,12 +300,14 @@ properties:
     name: 'friendlyName'
     description: A descriptive name for the dataset
     send_empty_value: true
-  - !ruby/object:Api::Type::KeyValuePairs
+  - !ruby/object:Api::Type::KeyValueLabels
     name: 'labels'
     description: |
       The labels associated with this dataset. You can use these to
       organize and group your datasets
-    default_from_api: true
+
+      **Note**: This field does not list all labels present on the resource in GCP.
```

Comment:
Here & others: Let's use the term "authoritative", since we use it in https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam (and all other IAM pages).

Something like: "This field is non-authoritative, and will only manage the labels present in your configuration. Please refer to the field `effective_labels` for all of the labels present on the resource.`

(`effective_labels` is output-only, so it's not entirely correct to call it authoritative)

Reply:
Yeah, this is a better and safer option.